app/controllers/guidance_controller.rb

```diff
@@ -18,4 +18,8 @@ def manually_registering_trainees
   def registering_trainees_through_hesa
     render(layout: "guidance_markdown")
   end
+
+  def check_data
+    render(layout: "guidance_markdown")
```

Comment:
```suggestion
    render(format: :md, layout: "guidance_markdown")
```
If we supply the format here AND remove the preceding underscore from the file name `_check_data.md`, then we can delete the view file `check_data.html.erb`.

Reply:
Cheers =)